addons/l10n_it_edi/models/res_partner.py

```diff
@@ -106,7 +110,7 @@ def _l10n_it_edi_get_values(self):
             zipcode = self.zip
             state_code = self.state_id and self.state_id.code
         else:
-            pa_index = 'XXXXXXX'
+            pa_index = '2R4GTO8' if is_sm else 'XXXXXXX'
```

Comment:
It looks too obscure to the casual reader. Adding a link to the referenced specification will do. Something like:

```
# San Marino is externally integrated with the SdI.
# The country as a whole has a single Destination Code.
# https://www.agenziaentrate.gov.it/portale/documents/20143/3788702/Modifiche+ProvvedimentonSanMarino+0248717-2021.pdf/429b5571-17b9-0cce-7f62-f79cf53086d7
```

Reply:
Thanks for the review. :slightly_smiling_face: 

I've added the comment.